x-pack/test/functional/services/rules/test_resources.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export function RulesTestResourcesServiceProvider({ getService }: FtrProviderContext) {
```

Comment:
I think we could move this provider to its own directory like:

`x-pack/test/functional/services/sample_data/test_resources.ts` I could see other plugins wanting to leverage creating and delete the kibana sample data and it's probably more intuitive to find that functionality in its own directory.

Reply:
Done in f5e1a38e153fbd01ca8328ef97784afb8ee5a2eb